Source/Events.Handling/EventHandlerAttribute.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dolittle.SDK.Events.Handling
+{
+    /// <summary>
+    /// Decorates a class to indicate the Event Handler Id of the Event Handler class.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public class EventHandlerAttribute : Attribute
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EventHandlerAttribute"/> class.
+        /// </summary>
+        /// <param name="eventHandlerId">The unique identifier of the event handler.</param>
+        /// <param name="partitioned">Whether the event handler is partitioned.</param>
+        /// <param name="scopeId">The scope that the event handler handles events in.</param>
+        public EventHandlerAttribute(string eventHandlerId, bool partitioned = true, string scopeId = default)
```

Comment:
How does this end up looking in the code if it is partitioned or scoped, I guess it means one could write
```csharp
[EventHandler("...", partitioned: false, scopeId: "...")].
```
Right?

I don't remember exactly what we agreed here, maybe we are missing something in the sketches. But I think it was _partitioned_ and _inScope_ in TS?

Reply:
That would be correct, it would look like you wrote there.

Yes, you are correct. I'll change it to inScope

Reply:
This should be resolved by latest commits